src/App.js

```diff
@@ -83,6 +84,7 @@ const DefaultLayout = ({ component: Component, ...rest }) => {
         </div>
         <NotifierComponent />
         <FAQContainer />
+        <CloudWalletUnlockContainer />
```

Comment:
I dont think this is a good place to put it.

Reply:
It gets into pages that never need to unlock cloud wallet.

Reply:
I suggest put it where it's needed, e.g. walletContainer, WalletSelectionContainer.